src/init.js

```diff
@@ -55,6 +55,11 @@ export const init = async (options) => {
 				return org.login;
 			});
 		});
+		let repos = await octokit.rest.repos.listForOrg({
+			org: 'searchspring',
+			type: 'public',
+			per_page: 500,
```

Comment:
Turns out the max for this request is 100...  😞 
https://octokit.github.io/rest.js/v18#:~:text=asc%2C%20otherwise%20desc-,per_page,Results%20per%20page%20(max%20100),-page

We will have to paginate to get them all with this endpoint... or maybe there is a better way?